<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>913</width>
    <height>472</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1001</width>
      <height>471</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="currentDateLbl">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>80</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Kartika</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(44,153,242);background:transparent;</string>
      </property>
      <property name="text">
       <string>Thursday, November 22nd</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="currentTimeColonLbl">
      <property name="geometry">
       <rect>
        <x>1098</x>
        <y>-4</y>
        <width>21</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Kartika</family>
        <pointsize>56</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(44,153,242);background:transparent;</string>
      </property>
      <property name="text">
       <string>:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="exitBtn">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="currentTimeHHLbl">
      <property name="geometry">
       <rect>
        <x>1010</x>
        <y>0</y>
        <width>99</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Kartika</family>
        <pointsize>56</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(44,153,242);background:transparent;</string>
      </property>
      <property name="text">
       <string>10</string>
      </property>
     </widget>
     <widget class="QLabel" name="currentTimeMMLbl">
      <property name="geometry">
       <rect>
        <x>1122</x>
        <y>0</y>
        <width>105</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Kartika</family>
        <pointsize>56</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(44,153,242);background:transparent;</string>
      </property>
      <property name="text">
       <string>22</string>
      </property>
     </widget>
     <widget class="QLabel" name="nameLbl">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>390</y>
        <width>791</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Kartika</family>
        <pointsize>40</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QVideoWidget" name="videoWidget" native="true">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>471</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="settingLbl">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>480</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>:/images/BossBoxSettings.png</pixmap>
      </property>
     </widget>
     <widget class="QPushButton" name="goBackBtn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>430</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Kartika</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(44,153,242);background:transparent;border:0px;</string>
      </property>
      <property name="text">
       <string>BACK</string>
      </property>
     </widget>
     <widget class="QLabel" name="firmwareVersionLbl">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>440</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Kartika</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(44,153,242);background:transparent;</string>
      </property>
      <property name="text">
       <string>FIRMWARE V1.3.1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header>qvideowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>goBackBtn</tabstop>
  <tabstop>exitBtn</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
